src/core/default_appearance.js

```diff
@@ -321,6 +321,18 @@ endcmap CMapName currentdict /CMap defineresource pop end end`;
       `/${this.fontName.name} ${numberToString(newFontSize)} Tf`,
     ];
 
+    const { resources } = this;
+    if (strokeAlpha !== 1) {
```

Comment:
Don't you need to check that it's actually a number as well, because in a corrupt PDF it could really be anything?